src/chrome/content/rules/PDFMerge.com.xml

```diff
@@ -0,0 +1,25 @@
+<!--
+	Non-functional hosts
+		SSL peer certificate was not OK:
```

Comment:
I have removed the comment accordingly. May I know how to test for this in general? thanks.